src/AceEditor/madie-ace-editor.tsx

```diff
@@ -95,9 +111,51 @@ const MadieAceEditor = ({
 
   const debouncedParse: any = useRef(
     _.debounce(async (nextValue: string) => {
+      //1. parse errors
       const errors = parseEditorContent(nextValue);
-      const annotations = mapParserErrorsToAceAnnotations(errors);
-      const aceMarkers = mapParserErrorsToAceMarkers(errors);
+
+      let allErrors: CqlError[] = [];
+      errors.forEach((parseError) => {
+        allErrors.push(parseError);
+      });
+
+      //2. ELM translation errors
+      const translationResults: ElmTranslation = await getElmTranslation(
```

Comment:
I believe that's not what we agreed on, Since for a large CQL, these calls may have a longer response time. So these calls are supposed to happen on a load of the Cql Editor tab and save.

Reply:
@RohitKandimalla Ok so debounce only does parse validation like before, no translation and vsac validation right?